README.md

```diff
@@ -1,87 +1,101 @@
+- [Transactions](https://godoc.org/github.com/bukalapak/redis#Multi).
+- [Pipeline](https://godoc.org/github.com/bukalapak/redis#example-Client-Pipeline) and [TxPipeline](https://godoc.org/github.com/bukalapak/redis#example-Client-TxPipeline).
+- [Scripting](https://godoc.org/github.com/bukalapak/redis#Script).
+- [Timeouts](https://godoc.org/github.com/bukalapak/redis#Options).
+- [Redis Sentinel](https://godoc.org/github.com/bukalapak/redis#NewFailoverClient).
+- [Redis Cluster](https://godoc.org/github.com/bukalapak/redis#NewClusterClient).
+- [Cluster of Redis Servers](https://godoc.org/github.com/bukalapak/redis#example-NewClusterClient--ManualSetup) without using cluster mode and Redis Sentinel.
+- [Ring](https://godoc.org/github.com/bukalapak/redis#NewRing).
+- [Instrumentation](https://godoc.org/github.com/bukalapak/redis#ex-package--Instrumentation).
 - [Cache friendly](https://github.com/go-redis/cache).
 - [Rate limiting](https://github.com/go-redis/redis_rate).
 - [Distributed Locks](https://github.com/bsm/redis-lock).
 
-API docs: https://godoc.org/github.com/go-redis/redis.
-Examples: https://godoc.org/github.com/go-redis/redis#pkg-examples.
 
 ## Installation
 
 Install:
 
 ```shell
-go get -u github.com/go-redis/redis
+go get -u github.com/bukalapak/redis
 ```
 
 Import:
 
 ```go
-import "github.com/go-redis/redis"
+import "github.com/bukalapak/redis"
 ```
 
 ## Quickstart
 
 ```go
 func ExampleNewClient() {
-	client := redis.NewClient(&redis.Options{
+	opts := &redis.Options{
+		CircuitBreaker: optCB,
 		Addr:     "localhost:6379",
 		Password: "", // no password set
 		DB:       0,  // use default DB
-	})
+	}
+
+	client := redis.NewClient(opts
 
 	pong, err := client.Ping().Result()
 	fmt.Println(pong, err)
 	// Output: PONG <nil>
 }
-
-func ExampleClient() {
-	err := client.Set("key", "value", 0).Err()
-	if err != nil {
-		panic(err)
+```
+### With Circuit Breaker
+```go
+func WithCircuitBreaker() {
+	
+	optCB := &hystrix.CommandConfig{
+		Timeout:                10000,
+		RequestVolumeThreshold: 2,
+		SleepWindow:            500,
+		ErrorPercentThreshold:  5,
 	}
+	// Please, read more about command config in hystrix-go doc.
+	//https://godoc.org/github.com/afex/hystrix-go/hystrix#pkg-variables)
 
-	val, err := client.Get("key").Result()
-	if err != nil {
-		panic(err)
-	}
-	fmt.Println("key", val)
-
-	val2, err := client.Get("key2").Result()
-	if err == redis.Nil {
-		fmt.Println("key2 does not exist")
-	} else if err != nil {
-		panic(err)
-	} else {
-		fmt.Println("key2", val2)
+	opts := &redis.Options{
+		CircuitBreaker: optCB,
+		Addr:     "localhost:6379",
+		Password: "", // no password set
+		DB:       0,  // use default DB
 	}
-	// Output: key value
-	// key2 does not exist
+
+	client := redis.NewClient(opts)
+
+	pong, err := client.Ping().Result()
+	fmt.Println(pong, err)
+	// Output: PONG <nil>
 }
-```
 
+```
+Any changes in breaker state will be generate metrics for monitoring. Below is a list of the metrics.
```

Comment:
state will be generate -> state will generate

Reply:
`will be` ini bukanya cotinous yah? soalnya kalo mereka ga pakai packen/promhttp metrics nya tidak ke generate. CMIWW

Reply:
will be + adjective/noun
will + verb